tests/test_mocks.py

```
@@ -0,0 +1,117 @@
+        )
+        
+        # Verify output file was created with correct content
+            assert output_file.exists()
+            output_content = output_file.read_text()
+            assert "Translated subtitle" in output_content
+            assert "00:00:01,000 --> 00:00:04,000" in output_content
+
+def test_progress_display_mocked():
+    """Test progress display with mocked console"""
+    with patch('rich.live.Live') as mock_live:
+        with patch('rich.console.Console') as mock_console:
```

Ruff (0.8.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)